frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -215,7 +491,7 @@ export function DynamicConfiguration () {
                     onClick={handleSubmit}
                     loading={state.formSubmitting}
                     data-cy='dynamic-configuration-submit'
-                    disabled={!isValid}
+                    disabled={state.formSubmitting}
```

Comment:
Why not both?
`!isValid || state.formSubmitting`

Reply:
Good idea, will do both